packages/idx/package.json

```diff
@@ -30,12 +30,12 @@
     "npm": ">= 8.19.3"
   },
   "dependencies": {
-    "@stratusjs/angular": "^0.12.0",
-    "@stratusjs/angularjs": "^0.10.0",
-    "@stratusjs/angularjs-extras": "^0.15.3",
+    "@stratusjs/angular": "^0.12.1",
+    "@stratusjs/angularjs": "^0.11.0",
+    "@stratusjs/angularjs-extras": "^0.16.0",
     "@stratusjs/boot": "^1.2.0",
     "@stratusjs/map": "^0.6.6",
-    "@stratusjs/runtime": "^0.12.3",
+    "@stratusjs/runtime": "^0.14.0",
```

Comment:
If you iterate the runtime to `0.14.0`, it will disable SSO in this version.

Reply:
alright, republish to only depend on 0.13.0. the versions don't exactly match with this